olive/common/hf/quant.py

```diff
@@ -161,9 +162,10 @@ def forward(
         group_size: int,
         in_features: int,
         out_features: int,
+        dynamo: bool = False,
     ):
         if torch.onnx.is_in_onnx_export():
```

Comment:
Instead, you may do 

```
if torch.onnx.is_in_onnx_export() and torch.compiler.is_exporting()
```

to avoid having to pass the dynamo parameter around. See https://docs.pytorch.org/docs/2.7/generated/torch.compiler.is_exporting.html